### DISCUSSION TITLE
Regarding ads on Floorp, as per the Ablaze GitHub page

### DISCUSSION THREAD

**Discussion by MMachado05** [Floorp-Projects/Floorp#1146] [Q&A] [answered]
I decided to hop back into the Floorp GitHub to see how development has been going, after the hype died down a little from a few months ago. One thing I noticed, that I don't believe was present in the past, was the existence of an "advertisement" tier in the GitHub sponsors page.

While I disagree with this decision, personally, I understand that finding consistent financial contributions can be difficult without some amount of compensation on the part of the developer, but I digress. What, exactly, would an advertisement on Floorp look like? Would it be in a little banner on your new tab page? Would it appear permanently in some side bar? Just curious as to how this would (hypothetically) affect the end user.

**Comment by CutterKnife** [comment, marked as answer]
Any company or organization that makes a certain amount of donations to Floorp on an ongoing basis may place an ad in Floorp. Currently this appears as a bookmark on the "new tab" page (which can be hidden if desired).

They are currently looking for ways to secure funding to continue development. This may include paid products and more types of advertising, but does not limit currently available functionality or invade privacy.
https://blog.ablaze.one/4314/2024-05-16/
